pyseq.py

```diff
@@ -733,7 +805,7 @@ def getSequences(source):
         items = sorted(source, key=lambda x: str(x))
     elif isinstance(source, str):
         if os.path.isdir(source):
-            items = sorted(glob(os.path.join(source, '*')))
+            items = sorted(glob(os.path.join(source, '*.*')))
```

Comment:
Hmm, not sure about this one. This assumes only files with a "." in the name are valid sequences, but that may not always be true. Perhaps this kind of thing should be configurable, however (see issue #5).